templates/default/src/main/liberty/config/server.xml

```diff
@@ -1,11 +1,23 @@
 <server description="Liberty server">
     <featureManager>
-        <feature>microProfile-3.3</feature>
+        <feature>servlet-4.0</feature>
+        <feature>cdi-2.0</feature>
+        <feature>jaxrs-2.1</feature>
+        <feature>jaxrsClient-2.1</feature>
+        <feature>jsonb-1.0</feature>
+        <feature>jsonp-1.1</feature>
+        <feature>mpFaultTolerance-2.1</feature>
+        <feature>mpMetrics-2.3</feature>
+        <feature>mpOpenAPI-1.1</feature>
+        <feature>mpHealth-2.2</feature>
+        <feature>mpConfig-1.4</feature>
+        <feature>mpOpenTracing-1.3</feature>
+        <feature>mpRestClient-1.4</feature>
     </featureManager>
-    <quickStartSecurity userName="admin" userPassword="adminpwd"/>
+    <!--  quickStartSecurity userName="admin" userPassword="adminpwd"/ -->
     <keyStore id="defaultKeyStore" location="key.jks" type="jks" password="mpKeystore"/>
-    <httpEndpoint host="*" httpPort="${default.http.port}" 
-        httpsPort="${default.https.port}" id="defaultHttpEndpoint"/>
+    <httpEndpoint host="*" httpPort="9080"
```

Comment:
I guess it's OK to take it out..we could add it back later...  

Another thing though would be to add defaultValue(s) in the pom.xml props:  e.g.  **liberty.defaultVar.default.http.port**.